packages/ui/src/components/Tooltip/Tooltip.tsx

```diff
@@ -40,12 +43,20 @@ export const Tooltip = forwardRef<HTMLDivElement, TooltipProps>(
     return (
       <AriaTooltip
         className={clsx(classNames.tooltip, className)}
+        offset={16}
```

Comment:
Using the offset property with a fixed value here since the height of the tooltip is fixed. The gap between the tooltip arrow and the target element is handled by the css margins, which use theme variables.

Reply:
I changed the value to use a default of `12px` that you can override if needed.